client/src/bin/ipc-client.rs

```diff
@@ -0,0 +1,14 @@
+use client::{cli, health_check, start_node, HEALTH_CHECK, NODE};
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+
+    let cmd = cli();
+    let matches = cmd.get_matches();
```

Comment:
We should figure out a cleaner and more general way of doing this so including new commands is as easy as implementing some kind of `RpcCmd` trait and providing it to an enum of available commands.